app/platform/templates/platform/developers/create_collection.html

```diff
@@ -0,0 +1,34 @@
+{% extends "platform/manage_grant_base.html" %}
+
+{% from "govuk_frontend_jinja/components/error-summary/macro.html" import govukErrorSummary %}
+{% from "govuk_frontend_jinja/components/back-link/macro.html" import govukBackLink %}
+
+
+{% block pageTitle %}
+    Setup a new collection - {{ grant.name }} - MHCLG Funding Service
+{% endblock pageTitle %}
+
+{% set active_sub_navigation_tab = "grant_developers" %}
+
+{% block beforeContent %}
+    {{ govukBackLink({
+        "text": "Back",
+        "href": url_for("platform.grant_developers_collections", grant_id = grant.id)
+    }) }}
+{% endblock beforeContent %}
+
+{% block content %}
+    <div class="govuk-grid-row">
+        <div class="govuk-grid-column-two-thirds">
+            <h1 class="govuk-heading-l">Set up a new collection</h1>
+            <form method="post" novalidate>
+                {{ form.csrf_token }}
+                {{ form.grant_id }}
+                {{ form.name }}
+                {{ form.submit(params={
+                    "text": "Set up collection"
+                }) }}
```

Comment:
Any reason we set the label in the template here rather than on the submit field of the form?

Reply:
I think because we use the same form for update or add and the text would be different for each

Reply:
Ahh ok ok